packages/react-super-components/src/components/msc-tile/msc-tile.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { Locale, defaultLocale } from '@ovhcloud/msc-utils';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { OsdsChip } from '@ovhcloud/ods-components/chip/react';
+import { OsdsTile } from '@ovhcloud/ods-components/tile/react';
+import { OsdsLink } from '@ovhcloud/ods-components/link/react';
+import { OsdsIcon } from '@ovhcloud/ods-components/icon/react';
+
+import { ODS_ICON_NAME, ODS_ICON_SIZE } from '@ovhcloud/ods-components/icon';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+
+interface Badge {
```

Comment:
```suggestion
export interface Badge {
```

Reply:
Why export it's only use in this file ?

Reply:
because when you use the component, you want to know the types of its props :

```
const badges: Badge[] = [{text: 'blabla', color: ODS_THEME_COLOR_INTENT.primary}]
<ScTile badges={badges} {...} />
```

And you should use `type` instead of `interface` since we don't use OOP in React. `interface`s won't be `implemented` by `class`es.

Reply:
For me we should not export interface that we will not use in other file to avoid exporting useless class
Even if it's not export the following code will trigger an error `text is missing in badges`

```
const badges = [{textError: 'blabla', color: ODS_THEME_COLOR_INTENT.primary}]

<ScTile badges={badges} {...} />

```
And for the interface VS type, following the exemple of React doc `https://react.dev/learn/typescript` they use interface instead of class

Reply:
"For me we should not export interface that we will not use in other file to avoid exporting useless class"
It's a good way to think when coding an app

But when it's a library we should export everything that is exposed to the user of the lib. If we build dynamically the list of badges and we create a function for this, it's better to be able to import the type directly instead of duplicating it in the code that uses the library :

```
const createBadgeList = (someParams, someParams2): Badge[] => {...};
```

Reply:
And concerning the debate about type vs interface : 
https://blog.logrocket.com/types-vs-interfaces-typescript/

I think we should use types because declaration merging can be dangerous in our case :
If you declare and export multiple times the same interface, they will merge and you will have a dynamically created interface (but you may not know where the other interface comes from and it can create conflicts). If you do the same with types, Typescript gives an error.

Additionally, types have some interesting features that interfaces don't have, as explained in the blog post.